app/store/modules/product/index.js

```diff
@@ -0,0 +1,35 @@
+// store/modules/product/index.js
+import productData from './mock';
+
+const namespaced = true;
+
+// initial state
+const state = {
+  items: productData // demo
```

Comment:
- [ ] I would recommend to name this `product` since, it will contain an object for a single product.

Reply:
- [ ] I would recommend that we also add documentation for properties, since most of them will be `null` or boolean initially when we handle real data.

```
const state = {
  /**
    * @type   {object}
    */
   product: productData
}
```